src/tests/test_michigan_fips.py

```diff
@@ -18,9 +18,11 @@ def get_arcgis_fipscode():
 
 def get_data_fipscode():
   file_ = os.path.join(public_dir, 'michigan.json')
-  with open(file_) as json_file:
-    rows = json.load(json_file)
-    return sorted([row['fipscode'] for row in rows])
+  if os.path.exists(file_):
+    with open(file_) as json_file:
+      rows = json.load(json_file)
+      return sorted([row['fipscode'] for row in rows])
+  return None
```

Comment:
Consider throwing an error if the file is not found.

Reply:
An error can prevent other, successful updates from getting pushed through. E.g., the webkit workflow which only runs Nevada (so michigan.json doesn't exist).

Reply:
Good call.